pkgs/development/python-modules/pylibjpeg-libjpeg/default.nix

```
@@ -8,47 +8,62 @@
   poetry-core,
   setuptools,
   numpy,
+  pydicom,
+  pylibjpeg-data,
+  pylibjpeg,
 }:
 
-buildPythonPackage rec {
-  pname = "pylibjpeg-libjpeg";
-  version = "2.3.0";
-  pyproject = true;
+let
+  self = buildPythonPackage rec {
```

```suggestion
let
  self = buildPythonPackage {
```
I would suggest to either do the `let self` trick or use `rec`, but not both?
For passthru.tests you could also add pylibjpeg-libjpeg as a parameter and use that.

---

`rec` removed.